### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by xh3b4sd**
This PR polishes the errgo package. See the updated readme for more information. I would like to use this in our projects. It does not look like the package is still maintained. So I would simply use our own fork. 

RFR @giantswarm/team-positive

### CODE REVIEW

File: errors/export_test.go [hunk 1]
```diff
@@ -1,6 +0,0 @@
-// Copyright 2014 Canonical Ltd.
```
Comment (hectorj2f) [inline errors/export_test.go:1]: and this test @xh3b4sd ?

**Review by hectorj2f — CHANGES_REQUESTED** [review]

File: errors/export_test.go (hunk 1 above)
Comment (xh3b4sd) [reply errors/export_test.go:1]: The `errors` package is deprecated. It should not have any relevance. See https://github.com/giantswarm/errgo/pull/1/files#diff-67bcaf0cce945ce4f52191db72018763L11.

Comment (hectorj2f) [reply errors/export_test.go:1]: k

**Review by hectorj2f — APPROVED** [review]

**Comment by hectorj2f** [comment]
@xh3b4sd LGTM if it works for you

**Pull request merged**